blogpost-drools-model-compiler.md

```diff
@@ -0,0 +1,116 @@
+# How to start contributing to Drools Executable Model
+
+## What is the Executable Model?
+
+The Executable Model is a new way to execute business rules in Drools which provides a much faster startup time. You can check out the details in the [Drools Documentation](https://docs.jboss.org/drools/release/7.55.0.Final/drools-docs/html_single/index.html) or in other blog posts such as [Mario's](https://blog.kie.org/2021/06/profiling-and-optimizing-executable-model-generation.html).
```

Comment:
I think we should not pitch/introduce the exec-model only as for "faster startup time". What about mention that is a java executable representation of the DRL (or other format) produced at compile time that reduce startup time and improve also runtime performance (as far as I remember)